templates/paragraph/paragraph--call-to-action.html.twig

```diff
@@ -0,0 +1,7 @@
+{% include '@components/call-to-action/call-to-action.twig' with {
+  'media': content.field_cta_media|field_value is not empty ? content.field_cta_media|field_value : false,
+  'heading': content.field_cta_heading|field_value is not empty ? content.field_cta_heading|field_value : false,
+  'body': content.field_cta_body|field_value is not empty ? content.field_cta_body|field_value : false,
+  'url': content.field_cta_link|field_value[0]['#url'] is not empty ? content.field_cta_link|field_value[0]['#url'] : false,
+  'text': content.field_cta_link|field_value[0]['#title'] is not empty ? content.field_cta_link|field_value[0]['#title'] : false,
```

Comment:
@kmonahan Would you mind weighing in on the best way to call url/title from a link field?  Also, do we need add any cache catching to this paragraph template?  If you have time, it might be worth you taking a look at the existing paragraph templates (hero, accordion, etc.) in Gesso 5.x and seeing if any adjustments are needed there for cache catching or field value calls.

Reply:
Ok based on what's in the Hero paragraph template, I swapped it to look in the `paragraph` object instead of the `content` render array. And that template doesn't have any caching checks/additions. But caching is never a bad thing to double-check, so yeah, is pulling from the paragraph ok? I feel like that's precisely when we'd need to add things to the cache list.

[7314900](https://github.com/forumone/gesso/pull/886/commits/731490071d152466f1c4f75784265772c2f47322)

Reply:
If we're pulling the URL and text values out of the field, then we need to ensure that cache tags and any attributes on the link itself are not lost. (I think the content array is also where we'd want to pull from, so that may need to get fixed in the hero paragraph too. 🤔 ) Agreed that this probably applies to other paragraphs as well. I'll take a look -- will be after various headless things though.